modules/op-release-notes-1-10.adoc

```diff
@@ -152,4 +152,17 @@ $ oc get tektoninstallersets.operator.tekton.dev | awk '/pipeline-main-static/ {
 * This update adds a conversion configuration to the `ResolutionRequest` CRD. This update properly configures conversion from the `v1alpha1.ResolutionRequest` request to the `v1beta1.ResolutionRequest` request.
 * This update checks for duplicate workspaces associated with a pipeline task.
 * This update fixes the default value for enabling resolvers in the code.
-* This update fixes `TaskRef` and `PipelineRef` names conversion by using a resolver.
\ No newline at end of file
+* This update fixes `TaskRef` and `PipelineRef` names conversion by using a resolver.
+
+[id="release-notes-1-10-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.1
+
+With this update, {pipelines-title} General Availability (GA) 1.10.1 is available on {product-title} 4.11 and later versions.
+
+[id="fixed-issues-1-10-1_{context}"]
+=== Fixed issues
+
+* Before this update, if the source branch information coming from payload included `refs/heads/` but the user-configured target branch only included the branch name, `main`, in a CEL expression, the push request would fail. With this update, {pac} passes the push request and triggers a pipeline if either the base branch or target branch has `refs/heads/` in the payload.
+* Before this update, when a `PipelineRun` object could not be created, the error received from the Tekton controller was not reported to the user. With this update, {pac} reports the error messages to the GitHub interface so that users can troubleshoot the errors. {pac} also reports the errors that occurred during pipeline execution.
+* With this update, {pac} does not echo a secret to the GitHub checks interface when it failed to create the secret on the {product-title} cluster.
```

Comment:
maybe add it that due of infrastructure issue or otherwise, or people think it's a bigger security issue than it is

Reply:
Incorporated